File: java_code/LISTTestScoring/src/TestIt.java

import java.io.BufferedWriter;
import java.io.FileWriter;
import java.io.IOException;
import java.util.logging.Level;
import java.util.logging.Logger;
import java.util.Random;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author andrej
 */
public class TestIt {

    public static void main(String[] args) {
        String encryptPhrase = randomString();
        System.out.println(encryptPhrase);
        try {
            BufferedWriter bw = new BufferedWriter(new FileWriter("__list_encrypt_phrase.txt"));
            bw.write(encryptPhrase);
            bw.close();
        } catch (IOException ex) {
            Logger.getLogger(TestIt.class.getName()).log(Level.SEVERE, null, ex);
        }
        LISTTestScoring lts = new LISTTestScoring();
        lts.setScore("Test", 10.0, 35.0);
        lts.updateScore("Test 2 \\ test 1", 12.0, 25.0);
        lts.setScore("\"test\"", 0.5, 40.0);
        lts.updateScore("\"test\"", 25.8, 40.0);
    }
    
    private static String randomString() {
        StringBuilder sb = new StringBuilder();
        
        Random r = new Random();
        String aplhabet = "ABCDEFGHIJKLMNOPRSQTUVWXYZ0123456789";
        
        for (int i = 0; i < 8192; i++) {
            sb.append(aplhabet.charAt(r.nextInt(aplhabet.length())));
        }
        
        return sb.toString();
    }
}

File: python_code/LISTTestScoring/TestIt.py
import random
import LISTTestScoring 

def randomString():
	chars = list()
	alphabet = list('ABCDEFGHIJKLMNOPRSQTUVWXYZ0123456789')
	print(alphabet)
	for i in range(0, 8192):
		index = random.randint(0, len(alphabet) - 1)
		chars.append(alphabet[index])
	return ''.join(chars)

f = open('__list_encrypt_phrase.txt', 'w')
phrase = randomString()
f.write(phrase)
f.close()
print(phrase)
print('\n')

l = LISTTestScoring.LISTTestScoring()
l.updateScore('Priklad 1', 5.0, 25.0)
l.updateScore('Priklad 1', 10.0, 25.0)
